env.go

```diff
@@ -152,6 +167,10 @@ func customOptions(opt Options) Options {
 }
 
 func optionsWithEnvPrefix(field reflect.StructField, opts Options) Options {
+	rawEnvVars := opts.rawEnvVars
+	if rawEnvVars == nil {
+		rawEnvVars = make(map[string]string)
```

Comment:
the test suite never enters inside this `if`, giving a coverage of  99.7% of the statements... 

but as we are creating a new Option, I think that it's important to make sure that rawEnvVars is not nil

Reply:
I think that, since we always use the `defaultOptions()` func, it should never be nil...